filament/backend/src/opengl/PlatformCocoaGL.mm

```diff
@@ -28,6 +28,7 @@
 #include <OpenGL/OpenGL.h>
 #include <Cocoa/Cocoa.h>
 
+#include <map>
```

Comment:
use unordered_map instead

Reply:
No longer applicable because of the refactor below.